src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -110,9 +110,19 @@ class AddressValidationModal extends React.Component {
     const isFirstOptionOrEnabled =
       (isAddressFromUser && validationKey) || !isAddressFromUser;
 
+    // calculate label height so input and label match in size
+    // this enables the entire label to be clickable
+    // 24 is the height of each span within the label
+    const labelHeight =
+      Object.values({
+        addressLine1,
+        addressLine2,
+        addressLine3,
+        city,
+      }).filter(val => Boolean(val)).length * 24;
```

Comment:
Is there no way to do this with CSS? :(

Reply:
There is! @erikphansen I jussst pushed changes to do that